core/src/io/github/teamfractal/entity/LandPlot.java

```diff
@@ -198,9 +198,9 @@ private synchronized boolean tryInstallRoboticon(Roboticon roboticon) {
      */
     public int[] produceResources() {
         int[] produced = new int[productionModifiers.length];
-        for (int i = 0; i < 2; i++) {
-            produced[i] = (int) ((float) productionAmounts[i] * productionModifiers[i]);
-        }
+        produced[0] = produceResource(ResourceType.ORE);
```

Comment:
Also needs a comment saying changed by JBT and that we did it to make it more maintainable